buildenv

```diff
@@ -29,3 +29,23 @@ expectedFailures:3
 ZZ
 }
 
+zopen_append_to_env()
+{
+cat <<EOF
+if [ "\${SSL_CERT_FILE}x" = "x" ]; then
+  if [ -f "\$PWD/cacert.pem" ]; then
+    export SSL_CERT_FILE="\$PWD/cacert.pem"
+  else
+    echo "Warning: you need to set SSL_CERT_FILE to a pem file if you want to use curl with SSH" >&2
+  fi
+fi
+EOF
+}
+
+zopen_append_to_setup() {
+cat <<EOF
+if [ -f "\$PWD/../cacert.pem" ]; then
```

Comment:
Were you still planning on making it a hidden file? 

If git is installed in `zopen/prod/git`, would it also need to be in `zopen/prod` for this to work?

Reply:
after thinking about this a bit i figured i had to stick the pem file from meta at the prod,boot,dev level vs one level higher, otherwise this wouldn't work for people that might need different pem files for git, curl and also if there was development.
Seemed like the lesser of two evils to have it at the 'global' level for just boot, prod, dev, /usr/local/zpmopentools

I decided against hidden but we could do that...